goldenboy.js

```diff
@@ -17,22 +17,41 @@ if (process.env.NODE_ENV === 'development') {
   require('dotenv').config();
 }
 
+const {integrationsValid} = require('./src/integrations')
+
+if(integrationsValid["trello"]) {
```

Comment:
Change integrationsValid["trello"] to integrationsValid.trello.  The `[""]` syntax for accessing fields is mostly used for accessing object fields dynamically with a variable